app/views/places/_form.html.erb

```diff
@@ -12,18 +12,18 @@
       </div>
     <% end %>
 
-    <div class="field">
+    <div class="form-group">
       <%= form.label Place.human_attribute_name("name") %>
-      <%= form.text_field :name %>
+      <input type="name" class="form-control" %>
```

Comment:
Do not remove rails helper methods just add to them needed classes, because your form will stop working correctly.

Reply:
<%= form.text_field(:name, class: "form-control" )%>